openmetadata-ui/src/main/resources/ui/src/utils/EntityUtilClassBase.ts

```diff
@@ -281,6 +281,17 @@ class EntityUtilClassBase {
       }
     }
   }
+
+  public getManageExtraOptions(entityType: string, _: string) {
+    switch (entityType) {
+      case EntityType.TABLE: {
+        return [];
+      }
+      default: {
+        return [];
+      }
+    }
```

Comment:
Instead of using switch can we change this to Object key approach, that will be more clear and efficient ?